data/abilities.ts

```diff
@@ -2716,7 +2716,7 @@ export const Abilities: {[abilityid: string]: AbilityData} = {
 	},
 	opportunist: {
 		onFoeAfterBoost(boost, target, source, effect) {
-			if (effect?.effectType !== 'Move') return;
+			if (effect && ['Opportunist', 'Mirror Herb'].includes(effect.fullname)) return;
```

Comment:
This failed because `effect.fullname` is `ability: Opportunist` or `item: Mirror Herb`. Did you mean `effect.name`?